src/misc/lv_style.c

```diff
@@ -92,30 +102,27 @@ bool lv_style_remove_prop(lv_style_t * style, lv_style_prop_t prop)
         if(old_props[i] == prop) {
             lv_style_value_t * old_values = (lv_style_value_t *)style->v_p.values_and_props;
 
-            uint16_t * new_props = &style->prop1;
-            lv_style_value_t * new_values = &style->v_p.value1;
-
-            if(style->prop_cnt > 2) {
```

Comment:
what's real change here?

Reply:
`&style->prop1` is illegal now because `prop1` is a bitfield, so the code was refactored to avoid that.